sql/src/test/java/io/crate/jobs/CountContextTest.java

```diff
@@ -62,4 +66,40 @@ public void testClose() throws Exception {
         future.setException(new UnknownUpstreamFailure());
         verify(callback, times(1)).onClose();
     }
+
+    @Test
+    public void testKill() throws Exception {
```

Comment:
rename to something more specific... testKillCancelsOperationFuture or something like that